.env.development

```diff
@@ -66,3 +66,6 @@ COOKIE_SECURE_SAME_SITE_NONE=false
 # next.config.js.
 NEXT_PUBLIC_URLS_BASE_PATH=/newtab
 NEXT_PUBLIC_URLS_USE_TRAILING_SLASH=true
+
+# Media Endpoint
+NEXT_PUBLIC_MEDIA_ENDPOINT=https://dev-tab2017-media.gladly.io
```

Comment:
i feel like it's easier to just make dev and qa both point to dev, or have qa also point to prod

Reply:
Having QA point to dev works for me.